test/test_regression_cli.py

```diff
@@ -54,17 +48,17 @@ def get_csv_md5(dataset, engines, tmpdir):
 
 def setup_module():
     """Update retriever scripts and cd to test directory to find data"""
-    os.chdir(os.path.dirname(os.path.abspath(__file__)))
+    os.chdir(rood_dir)
+    os.system('cp -r {0} {1}'.format(os.path.join(rood_dir, "test/raw_data"), rood_dir))
 
 
 def teardown_module():
     """Cleanup temporary output files after testing and return to root directory"""
-    os.chdir(os.path.dirname(os.path.abspath(__file__)))
+    os.chdir(rood_dir)
```

Comment:
In case tests fail/error in one module, we are sure that these module tests will begin from the right directory. 
This error used to be present where some modules error and the next module run errors if we do not set it to the right directory